ci/build-push-multiplatform.sh

```diff
@@ -82,33 +118,66 @@ function build_push() {
     esac
 
     local TAG
-    readonly TAG="${REPO_URL}:${BUILD_TAG}${BUILD_TYPE_SUFFIX}-${BUILD_ARCH}"
+    readonly TAG="${REPO_URL}:${BUILD_TAG}${BUILD_TYPE_SUFFIX}-${BUILD_PLATFORM}-${BUILD_ARCH}${BASE_IMAGE_TAG_SUFFIX}"
     local LATEST_TAG
-    readonly LATEST_TAG="${REPO_URL}:latest${BUILD_TYPE_SUFFIX}-${BUILD_ARCH}"
+    readonly LATEST_TAG="${REPO_URL}:latest${BUILD_TYPE_SUFFIX}-${BUILD_PLATFORM}-${BUILD_ARCH}${BASE_IMAGE_TAG_SUFFIX}"
 
+    # --provenance=false for docker buildx ensures that we create manifest instead of manifest list
     if [[ "${PUSH}" == true ]]; then
         echo "Building tags: ${TAG}, ${LATEST_TAG}"
-        docker buildx build \
-            --push \
-            --file "${DOCKERFILE}" \
-            --build-arg BUILD_TAG="${BUILD_TAG}" \
-            --build-arg BUILDKIT_INLINE_CACHE=1 \
-            --platform="${PLATFORM}" \
-            --tag "${LATEST_TAG}" \
-            --tag "${TAG}" \
-            .
+
+        if [[ "${BUILD_PLATFORM}" == "windows" ]]; then
+            docker build \
```

Comment:
Have we tried downloading the unattended installation of the buildx component from here?, looks like there are windows executables

https://github.com/docker/buildx?tab=readme-ov-file#manual-download

Binaries from the github release page
https://github.com/docker/buildx/releases/tag/v0.12.1

Reply:
oh nvm, looks like this is the root cause - https://github.com/docker/buildx/issues/176, buildx on windows containers not supported.

Reply:
Hmm, [maybe it is suppored](https://github.com/moby/buildkit/blob/master/docs/windows.md), but I would prefer to have more control over what happening with windows image builds and also avoid experimenting support. I would feel more comfortable to switch to it when it will be fully supported, or maybe if we go for FIPS/arm64